### ISSUE TITLE
gnu-efi building with the host toolchain

### ISSUE THREAD

**Issue by cdotstout** [effenel/bootimg#1]
Fails on mac where 'gcc' is clang.

gcc -I/Volumes/CaseSensitive/projects/fnljiri/out/unpack/gnu-efi-3.0.3/lib -I/Volumes/CaseSensitive/projects/fnljiri/out/unpack/gnu-efi-3.0.3/lib/../inc -I/Volumes/CaseSensitive/projects/fnljiri/out/unpack/gnu-efi-3.0.3/lib/../inc/x86_64 -I/Volumes/CaseSensitive/projects/fnljiri/out/unpack/gnu-efi-3.0.3/lib/../inc/protocol -mno-red-zone -mno-mmx -mno-sse  -g -O2 -fpic -Wall -fshort-wchar -fno-strict-aliasing -fno-merge-constants -ffreestanding -fno-stack-protector -fno-stack-check -DCONFIG_x86_64 -c /Volumes/CaseSensitive/projects/fnljiri/out/unpack/gnu-efi-3.0.3/lib/x86_64/callwrap.c -o x86_64/callwrap.o
/Volumes/CaseSensitive/projects/fnljiri/out/unpack/gnu-efi-3.0.3/lib/x86_64/setjmp.S:3:2: error: unknown directive
 .type setjmp, @function
 ^
/Volumes/CaseSensitive/projects/fnljiri/out/unpack/gnu-efi-3.0.3/lib/x86_64/setjmp.S:19:2: error: unknown directive
 .type longjmp, @function
 ^

**Comment by shinykitten** [comment]
FYI, this is fixed in https://fuchsia-review.googlesource.com/#/c/1472/ but pending verification from @cdotstout.  That verification is blocked on another issue that cropped up: effenel/kernel#2
